types/stylelint/index.d.ts

```diff
@@ -243,6 +243,135 @@ declare namespace stylelint {
 		meta?: RuleMeta;
 	};
 
+	/** @internal */
+	type BuiltInRules = {
+		readonly 'alpha-value-notation': Promise<Rule>;
```

Comment:
Should we also add this file to the developer guide for when adding a new rule?:

https://github.com/stylelint/stylelint/blob/main/docs/developer-guide/rules.md#wire-up-the-rule

Reply:
Good catch. Added with 5d45c6199.